packages/core/src/Stryker.ts

```diff
@@ -85,14 +90,18 @@ export default class Stryker {
       const testableMutants = await mutationTestProcessInjector
         .injectClass(MutantTestMatcher)
         .matchWithMutants(mutator.mutate(inputFiles.filesToMutate));
-      // IF OPT-IN
-      const statisticsProcess = inputFileInjector.provideValue('httpClient', new HttpClient('httpClient')).injectClass(Statistics);
+      let config = readConfig(new ConfigReader(this.options, this.log));
+      const statisticsProcess = inputFileInjector
```

Comment:
Deze `inputFileInjector` is bedoeld voor het gedeelte van Stryker wat zich bezighoudt met het inlezen en transpilen van files, door het toevoegen van `Statistics` is de verantwoordelijkheid hiervan opeens veel breder. Je kan 'm beter configureren in de [main injector](https://github.com/Jelle28/stryker/blob/master/packages/core/src/di/buildMainInjector.ts) zodat je 'm ook in de rest van de applicatie zou kunnen laten injecten